onTruckPlugin/src/main/java/com/github/ontruck/filters/FilterManager.java

```diff
@@ -0,0 +1,66 @@
+package com.github.ontruck.filters;
+
+import com.github.ontruck.MopedState;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import java.util.List;
+import java.util.LinkedList;
+
+public class FilterManager {
```

Comment:
I think this class could benefit from a explanation on how it should be used in relation to other parts of our plugin. Preferably javadoc.